boilerplates/shared-telefunc/files/components/TodoList.telefunc.ts

```diff
@@ -0,0 +1,9 @@
+import { db } from "@batijs/drizzle/database/db";
+import { todoTable, type TodoInsert } from "@batijs/drizzle/database/schema";
+import type { RunResult } from "better-sqlite3";
+
+export async function onCreateTodo({ text }: TodoInsert) {
+  const result: RunResult = await db.insert(todoTable).values({ text });
+
+  return { result };
+}
```

Comment:
Remove this, should be replace by **C**.

Reply:
This one is different from `+data.ts`. In `+data.ts`, we fetch all todos from the database, whereas this function is for inserting a new todo (with telefunc).

Reply:
I think later we can refactor {react|solid|vue}-telefunc, also merge `TodoList.telefunc.ts` of {react|solid|vue}-telefunc into this file and add conditions.